### PR TITLE
Replace alamofire errors

### LINKED ISSUE #55: Replace Alamofire errors

### PR DESCRIPTION
**Pull request by syllerim**
Closes #55

### CODE REVIEW

File: ApiConnector/APIConnectorError.swift [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+//  ApiConnector
+//
+//  Created by Mirellys on 29/03/2017.
+//  Copyright © 2017 WeAreReasonablePeople. All rights reserved.
+//
+
+import Foundation
+
+public enum APIConnectorError: Error {
+    case NoResponse
```
Comment (alickbass) [inline ApiConnector/APIConnectorError.swift]: all the cases should be lowercase

**Review by alickbass — CHANGES_REQUESTED** [review]
Please, make sure that all the cases of `enum` are lowercase. It's required since swift 3

File: ApiConnector/APIConnectorError.swift (hunk 1 above)
Comment (syllerim) [reply ApiConnector/APIConnectorError.swift]: 👍

File: ApiConnectorTests/APIConnectorErrorTests.swift [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+//  Created by Mirellys on 29/03/2017.
+//  Copyright © 2017 WeAreReasonablePeople. All rights reserved.
+//
+
+import XCTest
+import ApiConnector
+
+class APIConnectorErrorTests: XCTestCase {
+    
+    func testAPIConnectors() {
```
Comment (alickbass) [inline ApiConnectorTests/APIConnectorErrorTests.swift]: I would rename this method to something like `testApiConnectorErrorEquatable`

**Review by alickbass — CHANGES_REQUESTED** [review]

File: ApiConnector/APIConnectorError.swift [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+        case .unacceptableStatusCode(let code):
+            return "Status unnacceptable with code: \(code)"
+        }
+    }
+}
+
+extension APIConnectorError: Equatable {
+    public static func == (lError: APIConnectorError, rError: APIConnectorError) -> Bool {
+        switch (lError, rError) {
+        case (.noResponse, .noResponse):
```
Comment (alickbass) [inline ApiConnector/APIConnectorError.swift]: you can write this shorter: 
```swift 
case (.noResponse, .noResponse), (.invalidRequest, .invalidRequest):
    return true
```

**Review by alickbass — CHANGES_REQUESTED** [review]

**Review by alickbass — APPROVED** [review]

**Pull request merged**